devenv/docker/blocks/intentapi/Makefile

```
@@ -0,0 +1,75 @@
+KUBE_IPS       := 10.32.0.1,10.240.0.10,10.240.0.11,10.240.0.12,127.0.0.1
+KUBE_HOSTNAMES := localhost,kubernetes,kubernetes.default,kubernetes.default.svc,kubernetes.default.svc.cluster,kubernetes.svc.cluster.local
+
+CFG_DIR    := pki
+CERT_DIR   := certs
+CERT_CA    := $(CERT_DIR)/ca.pem $(CERT_DIR)/ca-key.pem
+CERT_SA    := $(CERT_DIR)/service-account.pem $(CERT_DIR)/service-account-key.pem
+CERT_KUBE  := $(CERT_DIR)/kubernetes.pem $(CERT_DIR)/kubernetes-key.pem
+CERT_ADMIN := $(CERT_DIR)/admin.pem $(CERT_DIR)/admin-key.pem
+
+all: certs config
+
+certs: $(CERT_CA) $(CERT_SA) $(CERT_KUBE) $(CERT_ADMIN)
+
+$(CERT_CA):
```

we could make this and the following target to depend on `install-cfssl` so then we could remove the additional step from the README
```suggestion
$(CERT_CA): install-cfssl
```